wrapanapi/systems/ec2.py

```diff
@@ -2,15 +2,10 @@
 from __future__ import absolute_import
 
 import base64
+import boto3
```

Comment:
is it already in requirements ?

Reply:
and please make sure that integration_tests requirements are updated accordingly.

Reply:
Yes, boto3 was used even before this PR:
https://github.com/ManageIQ/wrapanapi/blob/d4e7a0a551956dd36d9f04cf937c4865564e3f53/wrapanapi/systems/ec2.py#L11